### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by BasitKorede**
# ADD DESKTOP VERSION MY PORTFOLIO PROJECT

## This pull request adds the following

* Desktop version nav element such as portfolio, about, contact, with the use of media queries, etc.
* Desktop version background images for page attraction.
* recent work desktop section styled with the `css grid` and `css flexbox`
* Contact me section desktop version
  * the desktop version of the contact form element which has an additional input value of the last name.
  * the send button.

### Screenshot

<details>
    <summary>Click to view screenshot</summary>

screenshot](https://user-images.githubusercontent.com/69877769/154505689-76fdc079-4f71-456d-bc0b-0a33b685d66b.png)
)

</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -270,13 +408,21 @@ <h2 class="about-me-header">Software Developer</h2>
...
+        <div class="display-flex justify-content-center">
+          <img
+            src="./images/arrow-down.png"
+            class="arrow-down"
+            alt="arrow down"
+          />
+          <p class="work-paragraph">Work</p>
+        </div>
+
+        <ul class="social-media-ul display-flex justify-content-center">
```
Comment (elmar8287) [inline index.html:423]: - [ ] Please, consider fixing the space between the icons - it is too much. You can skip using `width` and it should display as needed.
Screen Shot 2022-02-17 at 13 09 47

File: index.html [hunk 2]
```diff
@@ -376,15 +510,40 @@ <h3 class="recent-work-article-header">Multi-post-story</h3>
...
+          <article class="recent-work-article display-none-sm display-block-lg">
+            <h3 class="recent-work-article-header">Multi-post-story</h3>
+            <p class="recent-work-paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul
+              class="recent-work-ul display-flex justify-content-space-between"
```
Comment (elmar8287) [inline index.html:523]: - [ ] Please, consider fixing the space between technologies - it is too much. You can fix it by removing the `    justify-content: space-between`
Screen Shot 2022-02-17 at 13 20 22

File: index.html [hunk 3]
```diff
@@ -376,15 +510,40 @@ <h3 class="recent-work-article-header">Multi-post-story</h3>
...
+              <li class="recent-work-ul-li">html</li>
+              <li class="recent-work-ul-li">boostrap</li>
+              <li class="recent-work-ul-li border-none">Ruby</li>
+            </ul>
+            <button
+              type="button"
+              aria-label="see project"
+              class="see-project-button border-none"
+            >
+              see projects
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider grammatical issues on the page like this, line 535 - fix it to "See project" 👍

File: index.html [hunk 4]
```diff
@@ -522,10 +728,17 @@ <h3 class="contact-me-header text-align">CONTACT US</h3>
...
             </button>
+            <button
+              type="submit"
+              aria-label="see project"
+              class="see-project-button see-project-button-form border-none display-none-sm display-block-lg"
+            >
+              Send
+            </button>
           </div>
         </form>
```
Comment (elmar8287) [inline index.html:754]: - [ ] Please, consider the design to be closer to Figma one. Here you can see on your version the space between the inputs.
- [ ] Please, consider the spaces between icons in the footer
> You version
Screen Shot 2022-02-17 at 14 15 48

> Figma design
Screen Shot 2022-02-17 at 14 17 01

File: index.html [hunk 5]
```diff
@@ -270,13 +408,21 @@ <h2 class="about-me-header">Software Developer</h2>
...
-        <ul class="social-media-ul display-flex">
+      <div
+        class="social-media-div display-flex-lg margin-0auto-lg justify-content-space-between"
+      >
+        <div class="display-flex justify-content-center">
+          <img
+            src="./images/arrow-down.png"
+            class="arrow-down"
+            alt="arrow down"
+          />
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that icon for scroll down should be one in mobile version. You have it doubled. I think it comes from the desktop version, where it should be doubled. 

Screen Shot 2022-02-17 at 14 33 58

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @BasitKorede, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  Check the comments under the review.
- [x]  Please, fix the screen displaying changes from 992px. In your version, it starts from 768px. It means, that the value for the `min-width` property should start from 992px




## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (BasitKorede) [reply index.html]: Thanks for the review @elmar8287 changes made 👍

File: index.html (hunk 5 above)
Comment (BasitKorede) [reply index.html]: Thanks, @elmar8287  changes made

File: index.html (hunk 1 above)
Comment (BasitKorede) [reply index.html:423]: Thanks, @elmar8287 changes made

File: index.html (hunk 2 above)
Comment (BasitKorede) [reply index.html:523]: Thanks for the review, @elmar8287 changes made

**Review by AnselemOdims — APPROVED** [review]
### Hi  @BasitKorede 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BasitKorede** [comment]
Hello 👋 @AnselemOdims  thanks for the review 😊

**Pull request merged**